src/utils/video-img_2.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+import shutil
+from pathlib import Path
+
+# global variable
+VIDEO_PATH = Path("D:/GDP/TESTING/nofi001.mp4")
+EXTRACT_FOLDER = Path("D:/GDP/TESTING/vi-im/")
+
+def extract_frames(video_path, result_path, index, desired_fps):
```

Comment:
use typing hints for your variables (you can look up `python typing hints` on google)